install_add_pod/after/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -41,9 +41,9 @@ Targets:
           ATTRIBUTES:
           - Project
     - Sources:
-      - JSONKit-dummy.m
       - JSONKit.m:
           COMPILER_FLAGS: "-fno-objc-arc"
+      - JSONKit-dummy.m
```

Comment:
@igor-makarov is this diff related to this PR https://github.com/CocoaPods/Xcodeproj/pull/839?

Reply:
its fine if it is by the way, just want to make sure I know where its coming from.

Reply:
verified that this was the reason. I updated to a SHA before this Pr was merged and regenerated integration specs and it did not include this diff.

will proceed with merging this.